packages/paste-core/utilities/grid/src/helpers.tsx

```diff
@@ -0,0 +1,71 @@
+import {ThemeShape} from '@twilio-paste/theme-tokens';
+import {ResponsiveValue} from 'styled-system';
+import {Margin, Space, SpaceOptions} from '@twilio-paste/style-props';
+import {Vertical} from '@twilio-paste/flex';
+import {
+  ColumnOffset,
+  ColumnOffsetOptions,
+  ColumnProps,
+  ColumnSpanOptions,
+  ColumnMinWidth,
+  ColumnWidthSpan,
+} from './types';
+
+export const getNegativeMargin = (theme: ThemeShape, gutter?: Space): Margin => {
```

Comment:
`getOuterGutterPull`? or something that describes why you need to do it?

Reply:
That works for me. It explains what it's doing in a better way.